src/org/labkey/test/pages/core/admin/ShowAdminPage.java

```diff
@@ -197,7 +197,7 @@ protected ElementCache newElementCache()
         protected WebElement analyticsSettingsLink = Locator.linkWithText("analytics settings").findWhenNeeded(this);
         protected WebElement externalRedirectHostLink = Locator.linkWithText("External Redirect Hosts").findElement(this);
         protected WebElement auditLogLink = Locator.linkWithText("audit log").findWhenNeeded(this);
-        protected WebElement authenticationLink = Locator.linkWithText("authentication").findWhenNeeded(this);
+        protected WebElement authenticationLink = Locator.linkWithText("authentication (deprecated)").findWhenNeeded(this);
```

Comment:
Remove this. Deprecated authentication page is no more.

Reply:
Good catch.  Changing it to no longer use the now-absent 'authentication (deprecated)' link, to use the one that does exist instead